### PR TITLE
ROf styling tweak

### PR DESCRIPTION
**Pull request by ssylver93**

### CODE REVIEW

**Review by yzlucas — APPROVED** [review]

File: client/wfnews-war/src/main/angular/src/app/components/report-of-fire/complex-question-page/rof-complex-question-page.component.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ::ng-deep.mat-button-toggle-label-content {
   line-height: 35px !important;
-  padding-bottom: 1.5rem !important
+  padding-bottom: 3% !important;
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/components/report-of-fire/complex-question-page/rof-complex-question-page.component.scss:5]: Are we sure a percentage is good here? Remember that mean it'll grow with screen size differences

Comment (ssylver93) [reply client/wfnews-war/src/main/angular/src/app/components/report-of-fire/complex-question-page/rof-complex-question-page.component.scss]: Changed to fixed size

File: client/wfnews-war/src/main/angular/src/app/components/report-of-fire/simple-question-page/rof-simple-question-page.component.scss [hunk 2]
```diff
@@ -13,7 +13,7 @@
         margin-bottom: 24px;
         border: none;
         border-radius: 8px;
-        height: 60px;
+        height: 80px;
```
Comment (yzlucas) [inline client/wfnews-war/src/main/angular/src/app/components/report-of-fire/simple-question-page/rof-simple-question-page.component.scss:16]: code conflict here, We may want to use 81px which is same as complex-questions buttons. 
Or use 80px on both.

**Review by yzlucas — CHANGES_REQUESTED** [review]

File: client/wfnews-war/src/main/angular/src/app/components/report-of-fire/simple-question-page/rof-simple-question-page.component.scss (hunk 2 above)
Comment (ssylver93) [reply client/wfnews-war/src/main/angular/src/app/components/report-of-fire/simple-question-page/rof-simple-question-page.component.scss]: Changed to 81px

**Review by yzlucas — APPROVED** [review]

**Pull request merged**